eya_def_tools/eya_def_tools/data_models/energy_assessment.py

```diff
@@ -19,11 +20,23 @@ class EnergyAssessmentResults(EyaDefBaseModel):
         min_length=1,
         description=(
             "Annual energy production (AEP) estimates at the turbine "
-            "location(s) in gigawatt hour (GW h). The dimension of the "
-            "first standard result dataset should be 'wind_farm_id'. "
-            "The dimension of the second standard result dataset "
-            "should be 'turbine_id'. Further results with other "
-            "dimensions may be included optionally."
+            "location(s) in gigawatt hour per annum (GW h year-1). The "
+            "dimension of the first standard result dataset should be "
+            "'wind_farm_id'. The dimension of the second standard "
+            "result dataset should be 'turbine_id'. Further results "
+            "with other dimensions may be included optionally."
+        ),
+    )
+    energy_production: Optional[list[Dataset]] = pdt.Field(
```

Comment:
This is to cover energy production results that are not annualised, such as the monthly production in a specific month.